---
title: Busca do modelo de AEM a partir do console do Campaign
description: Descrição
solution: Campaign,Experience Manager
product: Campaign,Experience Manager
applies-to: Campaign Classic,Experience Manager
keywords: KCS
resolution: Resolution
internal-notes: 'Ticket: E-000177400, E-000190218'
bug: false
article-created-by: Karan Khanna
article-created-date: 5/11/2022 11:32:36 AM
article-published-by: Karan Khanna
article-published-date: 5/11/2022 11:38:20 AM
version-number: 2
article-number: KA-16297
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=4a2fcd0a-1ed1-ec11-a7b5-00224809c556
exl-id: ffd904c5-ebe4-4490-bffe-deda0f64d47f
source-git-commit: e8f4ca2dd578944d4fe399074fab461de88ad247
workflow-type: tm+mt
source-wordcount: '188'
ht-degree: 1%

---

# Busca do modelo de AEM a partir do console do Campaign

## Descrição


Descrição do problema = Ao sincronizar AEM modelos para o Campaign, recebemos o seguinte erro:

INT-150012 A consulta HTTP retornou um erro do tipo &quot;Erro do Servidor Interno&quot; (500) A sessão atual não está autorizada a replicar /content/dam/xxx/support/images/others/ophone-x.png

Os seguintes erros são gerados nas camadas de AEM 1 e Campaign 2, respectivamente: 1 03.08.2020 11:21:12.517 \*AVISO\* qtp1229648141-8634 com.adobe.ams.environment.indicator.wcm.impl.AemEnvironmentIndicatorFilter não aceito para uri /mnt/overlay/granite/ui/content/shell/header/actions/pulse.data.json 03.08.2020 11:21:14.038 \*AVISO\* qtp1229648141-8819 com.adobe.ams.environment.indicator.wcm.impl.AemEnvironmentIndicatorFilter não aceito para uri /content/campaigns/xxx/mobile/xxx/samsung-note9.campai gn.link.json 03.08.2021 1:21:14.046 \*ERRO\* 10.44.3.4 1596453674038 POST /content/campaigns/xxx/mobile/xxx/samsung-note9.campaign.link.json HTTP/1.1 com.adobe.cq.mcm.campaign.servlets.LinkServlet Captura exceção ao servir link/solicitação de deslink java.lang.NullPop interException: null em com.adobe.cq.mcm.campaign.impl.NewsletterManagerImpl.link(NewsletterManagerImpl.java:113) com.day.cq.mcm.cq-mcm-campaign:1.3.14 em com.adobe.cq.mcm.campaign.servlets.LinkServlet.handlePost(LinkServlet.java:76) com.day.cq.mcm.cq-mcm campanha-campanha:1.3.14 em com.adobe.cq.mcm.campaign.servlets.LinkServlet.doPost(LinkServlet.java:50) com.day.cq.mcm.cq-mcm-campaign:1.3.14 em org.apache.sling.api.servlets.SlingAllMethodsServlet.mayService(SlingAllMethodsServlet.jlet ava:146) org.apache.sling.api:2.20.0

2 2020-08-05 05:56:17 Erro durante o processo de sincronização 2020-08-05 05:56:17 INT-150012 A consulta HTTP retornou um erro do tipo &quot;Erro Interno do Servidor&quot; (500) (iRc=-53)


## Resolução


Descrição do problema =



![](assets/3da0dec6-1ed1-ec11-a7b5-00224809c556.png)



Os direitos de replicação precisam ser concedidos ao usuário AEM do campaign, em AEM servidor.  Demos direitos de leitura/gravação/replicação ao usuário AEM da Campanha em /content/campaign e /content/dam paths onde os ativos da campanha estavam presentes.
